<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>802</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Выбор фигуры</string>
    </property>
    <addaction name="action_rectangle"/>
    <addaction name="action_oval"/>
    <addaction name="action_rhombus"/>
    <addaction name="action_line"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Палитра цветов</string>
    </property>
    <addaction name="action_choose_color"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_save"/>
    <addaction name="action_open"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_rectangle">
   <property name="text">
    <string>Прямоугольник</string>
   </property>
  </action>
  <action name="action_oval">
   <property name="text">
    <string>Овал</string>
   </property>
  </action>
  <action name="action_rhombus">
   <property name="text">
    <string>Ромб</string>
   </property>
  </action>
  <action name="action_choose_color">
   <property name="text">
    <string>Выбор цвета</string>
   </property>
  </action>
  <action name="action_line">
   <property name="text">
    <string>Линия</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="action_open">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
